src/editor-sidebar/save-panel.js

```diff
@@ -0,0 +1,130 @@
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+import {
+	// eslint-disable-next-line
+	__experimentalVStack as VStack,
+	// eslint-disable-next-line
+	__experimentalHeading as Heading,
+	// eslint-disable-next-line
+	__experimentalNavigatorToParentButton as NavigatorToParentButton,
+	PanelBody,
+	Button,
+	CheckboxControl,
+} from '@wordpress/components';
+import { chevronLeft, archive } from '@wordpress/icons';
+
+export const SaveThemePanel = () => {
+	const [ saveOptions, setSaveOptions ] = useState( {
+		saveStyle: true,
+		saveTemplates: true,
+		saveFonts: true,
+		removeNavRefs: false,
+		localizeText: true,
```

Comment:
I see how it might not be an option useful to everybody.  It also minimizes the use of patterns be default which I think is a good thing until that can be improved.

Reply:
I've defaulted it to false and also disabled it when saveTemplate is false:

<img width="278" alt="image" src="https://github.com/WordPress/create-block-theme/assets/146530/afcc4e2f-c9fd-4db4-a1ee-ea42af2afa9b">

Reply:
> It also minimizes the use of patterns be default which I think is a good thing until that can be improved.

True! Thanks, I think this is a better default setup.